packages/server/src/api/routes/application.ts

```diff
@@ -33,6 +34,16 @@ router
     authorized(BUILDER),
     controller.revertClient
   )
+  .post(
+    "/api/applications/publish",
```

Comment:
it's really rubbing me the wrong way how we don't use the app id in the path here while we do for the unpublish. Can we make them consistent and include the `appId` in both